src/helpers/shouldFail.js

```diff
@@ -0,0 +1,32 @@
+import chai from 'chai'
+
+const should = chai.should()
+
+async function shouldFailWithMessage (promise, message) {
```

Comment:
we never use `should` syntax in our tests. Why do you want to switch to should syntax?